lib/pester.rb

```diff
@@ -9,7 +9,7 @@ def self.configure(&block)
   end
 
   def self.retry(options = {}, &block)
-    retry_action(options.merge(on_retry: ->(_, delay_interval) { sleep(delay_interval) }), &block)
+    retry_action(options.merge(on_retry: Behaviors::Sleep::Constant), &block)
```

Comment:
yeah, that always was. it's really just `retry_action` with the defaults, plus "always sleep x seconds"